package.json

```diff
@@ -67,7 +67,7 @@
     "webpack": "1.12.13"
   },
   "peerDependencies": {
-    "cnvs": "1.1.3",
+    "cnvs": "~1.1.4",
```

Comment:
Yeah, I think we should keep the tilde, so it will not break `dcos-ui` as soon as we are out of sync with `reactjs-components`